### PR TITLE
FP-2956: Also update the drawers when active editor changes

### PR DESCRIPTION
**Pull request by manuelnogueiramov**
[FP-2956](https://movai.atlassian.net/browse/FP-2956)

* Removed duplication of call (`onFocus` and `onClick`) in the activation of the editor.
* Removed unused import.
* Added updateMenus to editor activation to reflect the update correctly on split view.
* Added activateTool to tools to have similar logic as the editors, and to eliminate some corner cases with split view.

[FP-2956]: https://movai.atlassian.net/browse/FP-2956?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by quirinpa — COMMENTED** [review]
This has to be well tested.. I don't think you need to remove onclick...
Test: changing in a split, moving to the other, changing it.
Click on original split. Press Ctrl+s.
Is the correct document saved?
If not. Please put onclick back.
Test on multiple document types.

**Comment by quirinpa** [comment]
The test I mentioned works fine. Please scratch what I've said.

**Review by quirinpa — DISMISSED** [review]

File: src/engine/ReactPlugin/ToolReactPlugin.js [hunk 1]
```diff
@@ -40,7 +55,15 @@ export function withToolPlugin(ReactComponent, methods = []) {
       };
     }, [off, on, profile.name]);
 
-    return <RefComponent {...props} ref={ref} />;
+    return (
+      <div
```
Comment (diasnad) [inline src/engine/ReactPlugin/ToolReactPlugin.js:59]: This div is causing an issue on the **Get Started** tab, it's not filling the component.
**Previously*
image
**Now**
image

**Review by diasnad — CHANGES_REQUESTED** [review]
On this PR we have a regression on the logs, on my side, I'm not receiving any log

image

File: src/engine/ReactPlugin/ToolReactPlugin.js (hunk 1 above)
Comment (manuelnogueiramov) [reply src/engine/ReactPlugin/ToolReactPlugin.js]: Oh damn, I didn't see that. 
The logs were actually there, but the div was not visible. Added the 100% height to the new container div. 

Nice catch

**Review by diasnad — APPROVED** [review]

**Review by quirinpa — APPROVED** [review]

**Pull request merged**